uk_geo_utils/helpers.py

```diff
@@ -242,9 +243,7 @@ def _get_addressable_object(
         elif start_suffix and end_number:
             ao = ao + start_suffix
 
-        if end_suffix and end_number:
-            ao = ao + "-"
-        elif start_number and end_number:
+        if (end_suffix and end_number) or (start_number and end_number):
```

Comment:
ruff auto-generated the line

`if end_suffix and end_number or start_number and end_number:`

in 8e7079d by collapsing this redundant `elif`

Ruff is correct that `and` has a higher order of precedence than `or`, but I find this a lot easier to parse quickly with brackets.